integrations/great_expectations/Great_Expectations_Hopsworks_Concepts.ipynb

```diff
@@ -0,0 +1,1568 @@
+        "\n",
+        "During development or prototyping, Great Expectations offers a DataFrame wrapper to explore your data. It enables auto-completion method for core supported expectations. As different expectations will require different kwargs for their configuration, it helps gaining familiarity with new expectations.\n",
+        "\n",
+        "In Hopsworks, expectations will usually be evaluated on a Feature or pair of Feature. The expectation type enables a standardisation of the metrics computed while the configuration allows the user to adapt the success of the validation to their particular conditions. In a production setup, each Feature will be validated by multiple expectations. Note that Hopsworks populates the meta field of expectation with an id to enable smoother integrations."
+      ],
+      "metadata": {
+        "id": "M77KYBpSqNYj"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "ge_df = ge.from_pandas(trans_df)\n",
+        "result = ge_df.expect_column_mean_to_be_between(column=\"amount\", min_value=10, max_value=100)\n",
+        "print(f\"Observed mean : {result['result']['observed_value']}, success : {result['success']}\")"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "ixoYH-wfptAE",
+        "outputId": "21006e84-d169-4786-e7b4-4b4c656d2d08"
+      },
+      "execution_count": 11,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Observed mean : 421.1216635540464, success : False\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Validation Result"
+      ],
+      "metadata": {
+        "id": "exL_EWitvkoZ"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "An expectation generates a validation result entity on evaluation. This result is specific to a DataFrame and acts as a log for the evaluation. There are different information in the validation result:\n",
+        "- The success or failure of the evaluation, meaning are the metrics calculated with the success criteria specified in the expectations.\n",
+        "- A meta field for extra information\n",
+        "- An expectation_config field which contains the entity that was evaluated against the DataFrame. Crucial for reproducibility, still needs the same data.\n",
+        "- An exception_info, in case the evaluation raised an exception.\n",
+        "- The result field itself, with information about the computed metrics. The type of information depends on the expectation_type.\n",
```

Comment:
Isn't there also the setting of how detailed the result is going to be? "complete" vs "basic", should we mention that?

Reply:
It is in the expectation config part, might be better to mention it in the expectation section if you want to add the info. It is tricky to find the right level of details to not overload the user, not sure if it is among the most needed info.

Reply:
ok we can omit it, power users will know